src/components/AppBar/AppBar.jsx

```diff
@@ -81,22 +86,48 @@ const AppBar = ({
           </ListItemText>
         </ListItem>
       );
-    })
-  );
+    });
+  // eslint-disable-next-line
+  const NavIcons = () => {
```

Comment:
What is the lint issue here? If there is a specific issue we want to avoid let's just add that `eslint-disable-next-line no-undef` that way if we find ourselves disabling a rule too many times we can ignore it at the project level

Reply:
I removed it and it works fine! I was having issues with my formatter & the linter but I think I've addressed them.